### DISCUSSION TITLE
Jetbrains IDE plugin

### DISCUSSION THREAD

**Discussion by DenuxPlays** [Orange-OpenSource/hurl#2784] [Ideas] [answered]
Hey,
a plugin to add support .hurl file inside jetbrain ides would be great.
Currently it's just a simple text file without any highlighting etc.

For now is there an hurl editor or something like that?

**Comment by jcamiel** [comment, marked as answer]
Hi @DenuxPlays,

Good IDE supports (mainly VSCode and IntelliJ) are our on roadmap (high priority)! There is a (non official) plugin for NeoVim => https://github.com/jellydn/hurl.nvim

**Reply by DenuxPlays** [reply]
Thank you :)
